src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -141,6 +141,9 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 	@Parameter(defaultValue = "${project.version}", readonly = true)
 	private String projectVersion;
 
+	@Parameter(property = "helm.namespace")
```

Comment:
Why not `default` as defaultValue?

Reply:
Move them to Abstract Class, but do not remove old k8s config. Log and replacement should be done in next major version.

Reply:
Done.